presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionProcess.java

```diff
@@ -263,4 +262,29 @@ private String getProcessWorkingPath(String path)
 
         return absolutePath.getAbsolutePath();
     }
+
+    private List<String> getLaunchCommand() throws IOException
+    {
+        String executablePath = getProcessWorkingPath(SystemSessionProperties.getNativeExecutionExecutablePath(session));
+        String programArgs = SystemSessionProperties.getNativeExecutionProgramArguments(session);
+        String configPath = Paths.get(getProcessWorkingPath("./"), String.valueOf(port)).toAbsolutePath().toString();
+        ImmutableList.Builder<String> command = ImmutableList.builder();
+        List<String> argsList = Arrays.asList(programArgs.split(" "));
+        boolean etcDirSet = false;
+        for (int i = 0; i < argsList.size(); i++) {
+            String arg = argsList.get(i);
+            if (arg.equals("--etc_dir")) {
+                etcDirSet = true;
+                configPath = argsList.get(i + 1);
+                break;
+            }
+        }
+        command.add(executablePath).addAll(argsList);
+        if (!etcDirSet) {
+            command.add("--etc_dir").add(configPath);
+            populateConfigurationFiles(configPath);
+        }
+        log.info("Launching %s \nConfig path: %s\n", executablePath, configPath);
```

Comment:
do we want to log out the full launch command instead of only the path and the config path?

Reply:
Good point. Yeah I think we should log the full command